ceph/rados/core_workflows.py

```diff
@@ -4615,3 +4616,56 @@ def get_rados_df(self, pool_name: str = None):
         out = self.run_ceph_command(cmd=_cmd, client_exec=True)
 
         return out["pools"][0] if pool_name else out
+
+    def set_service_managed_type(self, service_type, unmanaged) -> bool:
+        """
+        Method to set the service to either managed or unmanaged
+        Args:
+            unmanaged: True or false, for the service management
+            service_type : service types are- mon,mgr,osd,rgw, mds
+        returns:
+            Pass -> True, Fail -> false
+        """
+        cmd_export = f"ceph orch ls {service_type} --export"
+        out = self.run_ceph_command(cmd=cmd_export, client_exec=True)[0]
```

Comment:
We should be retrieving all the items in list.
Eg : 

```# ceph orch ls osd --export -f json-pretty

[
  {
    "placement": {
      "label": "osd"
    },
    "service_id": "osds_colocated",
    "service_name": "osd.osds_colocated",
    "service_type": "osd",
    "spec": {
      "data_devices": {
        "paths": [
          "/dev/ssd-vg/ssd-lv1"
        ]
      },
      "filter_logic": "AND",
      "objectstore": "bluestore"
    }
  },
  {
    "placement": {
      "label": "osd"
    },
    "service_id": "osds_noncolocated",
    "service_name": "osd.osds_noncolocated",
    "service_type": "osd",
    "spec": {
      "data_devices": {
        "paths": [
          "/dev/hdd-vg/hdd-lv1",
          "/dev/hdd-vg/hdd-lv2",
          "/dev/hdd-vg/hdd-lv3"
        ]
      },
      "db_devices": {
        "paths": [
          "/dev/nvme-vg/nvme-lv1",
          "/dev/nvme-vg/nvme-lv2",
          "/dev/nvme-vg/nvme-lv3"
        ]
      },
      "filter_logic": "AND",
      "objectstore": "bluestore"
    },
    "unmanaged": true
  }
] ```

Reply:
each service needs to be set to unmanaged. eg : 

```out = json.loads(osd_orch_op)

for outr1 in out:
    outr1.setdefault("unmanaged", True)
    print(outr1)```

Reply:
Included the code as per comments.